src/components/Modal/Modal.vue

```diff
@@ -525,6 +571,24 @@ export default {
 				this.slideshowTimeout.clear()
 			}
 		},
+		/**
+		 * Add focus trap for accessibility.
+		 */
+		useFocusTrap() {
+			const contentContainer = this.$refs.mask
+			this.$nextTick(function() {
```

Comment:
Do you know why it wasn't working without it? Could be nice to add a comment :)

Reply:
added